prototypes/entity.lua

```diff
@@ -1,302 +1,148 @@
-			result.inventory_size = 2000;
-			result.logistic_slots_count = 1;
-		end
-		if ((logistics_name == "requester") or (logistics_name == "buffer")) then
-			result.logistic_slots_count = 12;
-		end
-	end
-	return result;
-end
-
-function define_storehouse(name, logistics_name)
-	local entity_type = "container";
-	if (logistics_name ~= nil) then
-		entity_type = "logistic-container";
-	end
-	local result = 
-	{
-		type = entity_type,
-		name = "storehouse-" .. name,
-		icon = "__Warehousing__/graphics/icons/storehouse-" .. name .. ".png",
-		icon_size = 32,
-		flags = {"placeable-neutral", "placeable-player", "player-creation"},
-		minable = {mining_time = 2, result = "storehouse-" .. name},
-		max_health = 250,
-		corpse = "big-remnants",
-		dying_explosion = "medium-explosion",
-		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
-		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
-		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
-		scale_entity_info_icon = true,
-		resistances =
-		{
-			{
-				type = "fire",
-				percent = 90
-			}
-		},
-		collision_box = {{-1.2, -1.2}, {1.2, 1.2}},
-		selection_box = {{-1.5, -1.5}, {1.5, 1.5}},
-		fast_replaceable_group = "container",
-		inventory_size = 150,
-		scale_info_icons = true,
-		picture =
-		{
-			filename = "__Warehousing__/graphics/entity/storehouse-" .. name .. ".png",
-			priority = "high",
-			width = 129,
-			height = 100,
-			shift = {0.421875, 0},
-		},
-		-- circuit_connector_sprites = define_storehouse_connector({0.1875, 0.15625}),
-		circuit_wire_max_distance = 7.5,
-		circuit_wire_connection_point =
-		{
-			shadow =
-			{
-				red = {.26, -0.6},
-				green = {.36, -0.6}
-			},
-			wire =
-			{
-				red = {-0.16, -0.9},
-				green = {0.16, -0.9}
-			}
-		},
-	};
-
-	if (logistics_name ~= nil) then
-		result.minable.hardness = 0.2;
-		result.logistic_mode = logistics_name;
-		if (logistics_name == "storage") then
-			result.inventory_size = 300;
-			result.logistic_slots_count = 1;
-		end
-		if ((logistics_name == "requester") or (logistics_name == "buffer")) then
-			result.logistic_slots_count = 12;
-		end
-	end
-	return result;
-end
-
-data:extend(
-{
-	define_warehouse("basic", nil),
-	define_warehouse("passive-provider", "passive-provider"),
-	define_warehouse("storage", "storage"),
-	define_warehouse("active-provider", "active-provider"),
-	define_warehouse("requester", "requester"),
-	define_warehouse("buffer", "buffer"),
-
-	define_storehouse("basic", nil),
-	define_storehouse("passive-provider", "passive-provider"),
-	define_storehouse("storage", "storage"),
-	define_storehouse("active-provider", "active-provider"),
-	define_storehouse("requester", "requester"),
-	define_storehouse("buffer", "buffer"),
-})
-
+--[[ Copyright (c) Optera 2017
+ * Part of the warehousing mod
+ *
+ * See LICENCE in the project directory for licence information

```

Comment:
This is old. The current license header line refers to a different file.
```suggestion
 * See License.txt in the project directory for license information.
```